ios/LayoutReanimation/REASharedTransitionManager.m

```diff
@@ -277,6 +288,10 @@ - (NSArray *)sortViewsByTags:(NSArray *)views
       _snapshotRegistry[viewTarget.reactTag] = targetViewSnapshot;
     } else {
       targetViewSnapshot = _snapshotRegistry[viewTarget.reactTag];
+      if (targetViewSnapshot == nil) {
+        targetViewSnapshot = [[REASnapshot alloc] initWithAbsolutePosition:viewTarget];
+        NSLog(@"[Reanimated] Unable to find view style snapshot. It looks like you try to animate an unmounted view.");
```

Comment:
is this an error? Seem like we have a recovery path here so the question is if we really need/want to signal this to the developer? If yes, it'd be better to make a warning as this type of log will likely not be noticed.